05.ls/ls.rb

```diff
@@ -81,9 +81,7 @@ def display_files_info(block_size, files)
     print file[:user].rjust(7)
     print file[:group].rjust(7)
     print file[:size].to_s.rjust(6)
-    print file[:mtime].month.to_s.rjust(3)
-    print file[:mtime].day.to_s.rjust(3)
-    print file[:mtime].strftime('%H:%M').to_s.rjust(6)
+    print file[:mtime].strftime('%_m %_d %H:%M').to_s.rjust(12)
```

Comment:
```suggestion
    print file[:mtime].strftime(' %_m %_d %H:%M')
```

一番左にスペースを入れるだけならこれでいけそうです。

少なくとも`strftime`の戻り値は文字列なので`to_s`は不要ですね。